src/pages/better-dev-onboarding.tsx

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable unicorn/prevent-abbreviations */
+import { FunctionComponent } from 'react'
+
+import Link from 'next/link'
+
+import {
+    BlogResourceItem,
+    ContentSection,
+    CustomerLogos,
+    FeatureWalkthrough,
+    IntegrationsSection,
+    Layout,
+} from '@components'
+import { buttonStyle, buttonLocation } from '@data'
+import { useHubSpot, useChiliPiper } from '@hooks'
+
+import styles from './use-cases/useCases.module.scss'
+
+const resourceItems = [
+    {
+        title: 'Continuous developer onboarding: A guide to cultivating a culture of professional growth',
+        description:
+            'Download the guide to developer onboarding and learn how to shift to a culture of continuous onboarding in your engineering organization.',
+        type: 'Guide',
+        img: {
+            src: 'blog/thumbnails/continuous-developer-onboarding.png',
+            alt: 'Continuous developer onboarding guide thumbnail',
+        },
+        href: '/guides/continuous-developer-onboarding',
+    },
+    {
+        title: 'Nutanix fixed Log4j quickly and confidently with Sourcegraph',
+        description:
+            'See how Nutanix was able to confidently indentify every instance of Log4j across its sprawling codebase and deliver patches to its customers that fully remediated the vulnerability within 4 days.',
+        type: 'Case study',
+        img: {
+            src: 'https://sourcegraphstatic.com/blog/log4j/log4j-blog-thumbnail.png',
+            alt: 'Log4j Log4Shell 0-day blog thumbnail',
+        },
+        href: '/blog/log4j-log4shell-0-day',
+    },
+]
+
+const BetterDeveloperOnboarding: FunctionComponent = () => {
+    useHubSpot({
+        portalId: '2762526',
+        formId: '5b81cde2-4701-425e-8782-dc79d8979b70',
+        region: 'na1',
+        targetId: 'topForm',
```

Comment:
Stakeholders decided to use a ChiliPiper-enabled address form on the hero instead of the CTAs proposed in the spec: [Slack discussion](https://sourcegraph.slack.com/archives/C03H84VNQLC/p1654098074371459). Please note the updated design ^